### ISSUE TITLE
emails users in circles ? 

### ISSUE THREAD

**Issue by livier** [nextcloud/circles#1248]
Testing emails as users in a circle.
Putting some testings adress in “Add members” (“ajouter des membres” - in french)
Did not see anything happening,
no email received with a link.
How the recipient could identify and participate in circle's share ? 
Is there anything I missed or did not understood ?

**Comment by livier** [comment]
up^

**Comment by Uatschitchun** [comment]
Yes, what's this adding email users about? Right now nothing happens when adding an email address (no mail sent, no invitation)!?
As collectives use circles for member management, I thought it would be possible to use email addresses in circles to invite users not on the cloud to participate...🤔

**Comment by wunschadresse** [comment]
I understood it differently: You can add e-mails of people who will most definitely not be on that instance. to me that makes, working in a volunteer organization, where helping hands are only "loosely" connected to the organization. I still want to see them and have the ability to contact everybody via email.
